tests/algorithms/test_bc.py

```diff
@@ -212,6 +212,61 @@ def test_that_bc_improves_rewards(
     )
 
 
+def test_gradient_accumulation(
+    cartpole_venv: vec_env.VecEnv,
+    rng,
+    pytestconfig,
+):
+    demonstrations = make_expert_transition_loader(
+        cache_dir=pytestconfig.cache.makedir("experts"),
+        batch_size=6,
+        expert_data_type="transitions",
+        env_name="seals/CartPole-v0",
+        rng=None,
+        num_trajectories=17,
+    )
+
+    seed = rng.integers(2**32)
+
+    th.manual_seed(seed)
+    bc_trainer1 = bc.BC(
```

Comment:
A lot of code duplication between setup of `bc_trainer{1,2}` and subsequent training -- perhaps move this into a helper method?

Reply:
I'm not sure if it meaningfully can since it's just a single function call each time

Reply:
It's also `th.manual_seed(seed)` and the function call is quite long, I think it can improve readability, I'll make a commit to show what I mean

Reply:
Makes sense, your change looks good